src/Bicep.LangServer/Bicep.LangServer.csproj

```diff
@@ -1,45 +1,47 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <RootNamespace>Bicep.LanguageServer</RootNamespace>
-    <!-- generates a NOTICE file in the publish output using our custom targets -->
-    <EnableNoticeInPublishOutput>true</EnableNoticeInPublishOutput>
-  </PropertyGroup>
+	<PropertyGroup>
```

Comment:
[nit] Can you revert the indentation changes so that it's consistent with other project files (2 spaces)?

Reply:
Seems this file can be reverted.

Reply:
its just a hackathon branch so don't worry about it